knodle/trainer/trainer.py

```diff
@@ -79,6 +80,19 @@ def initialise_optimizer(self):
 
 class BaseTrainer(Trainer):
 
+    def __init__(
+            self,
+            model: Module,
+            mapping_rules_labels_t: np.ndarray,
+            model_input_x: TensorDataset,
+            rule_matches_z: np.ndarray,
+            **kwargs):
+        if kwargs.get("trainer_config", None) is None:
+            kwargs["trainer_config"] = BaseTrainerConfig()
+        super().__init__(model, mapping_rules_labels_t, model_input_x, rule_matches_z, **kwargs)
+
+        check_other_class_id(self.trainer_config, self.mapping_rules_labels_t)
```

Comment:
I think then we other_class_id is always set?
But shouldn't "None" mean that there is no other class id?